src/preact.d.ts

```diff
@@ -1,5 +1,5 @@
 declare namespace preact {
-	interface ComponentProps<C extends Component<any, any> | FunctionalComponent<any>> {
+	interface ComponentProps<C extends Component<PropsType, any> | FunctionalComponent<PropsType>, PropsType> {
```

Comment:
As PropsType isn't used in this interface and isn't constrained in any way, it provides no more safety than simply any, and only adds complexity.